qsiprep/workflows/dwi/merge.py

```diff
@@ -149,13 +149,13 @@ def init_merge_and_denoise_wf(
 
     # Get a data frame of the raw_dwi_files and their imaging parameters:
     dwi_df = get_acq_parameters_df(raw_dwi_files, layout=layout)
-    for dwi_num, row in dwi_df.iterrows():
-        dwi_num += 1  # start at 1
+    for i_dwi, row in dwi_df.iterrows():
+        dwi_num = i_dwi + 1  # start at 1
```

Comment:
I just changed this because modifying a loop variable in the loop is considered bad practice. I don't actually think it would have any effect here.